.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -1,46 +1,53 @@
-name: Levitate
+name: Levitate / Report breaking changes
 
-on: push
+on:
+  workflow_run:
+    workflows: ["Levitate / Detect breaking changes"]
+    types: [completed]
 
 jobs:
-  build:
-    name: Detecting breaking changes
+  notify:
+    name: Report
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Install Node.js
+    - name: Setup environment
       uses: actions/setup-node@v2
       with:
-        node-version: '16'
+        node-version: 16
```

Comment:
nit: All the examples put version in single quotes, but I am pretty sure it will work either way. Example https://github.com/actions/setup-node/blob/main/README.md?plain=1#L24